sites/mosul/configs/openmrs/initializer_config/ampathformstranslations/F13-PNC_translations_ar.json

```diff
@@ -4,89 +4,177 @@
+    "Abdominal surgery": null,
+    "Abnormal": null,
+    "Abnormal bleeding": null,
     "Abortion": "إجهاض",
     "Additional remarks": "ملاحظات إضافية",
+    "Alive": null,
+    "Anaemia": null,
     "Anaemia (conjunctiva, heamoglobin)": "فقر الدم (الملتحمة، الهيموجلوبين)",
+    "Antepartum hemorrhage": null,
     "Any of selected items should also be added in 'Conditions'": "يجب أيضًا إضافة أي من العناصر المحددة في الشروط",
     "Appearance (colour, breathing, activity)": "المظهر (اللون، التنفس، النشاط)",
+    "Birth spacing and family planning": null,
     "Birth weight (g)": "وزن الولادة (جم)",
+    "Breast / Breastfeeding problem": null,
+    "Breastfeeding and formula": null,
+    "Breastfeeding only": null,
+    "Breastfeeding support": null,
     "Breasts (e.g. infection, enforgement)": "الثديين (على سبيل المثال العدوى، الانضغاط)",
+    "COC": null,
+    "Caesarean section": null,
     "Check in 'Immunizations' for Hepatitis B and other birth dose vaccines": "تحقق في التطعيمات الخاصة بلقاح التهاب الكبد B واللقاحات الأخرى التي يتم تناولها عند الولادة",
     "Check in 'Immunizations' for tetanus vaccine": "التحقق من التطعيمات للقاح الكزاز",
     "Check in 'Medications' for Tetracycline eye ointment and Vitamin": "تحقق في الأدوية من مرهم العين تيتراسايكلين وفيتامين",
     "Check in 'Results Viewer' for latest laboratory test results": "قم بالتحقق من عارض النتائج للحصول على أحدث نتائج الاختبارات المعملية",
-    "Check in 'Vital & Biometrics' and fill with updated measurements for 'Blood pressure', 'Temperature' and 'MUAC'": "التأكد من العلامات الحيوية والبيومترية  - ومليء القياسات المحدثة لضغط الدم",
+    "Check in 'Vital & Biometrics' and fill with updated measurements for 'Blood pressure', 'Temperature' and 'MUAC'": "التأكد من العلامات الحيوية والبيومترية - ومليء القياسات المحدثة لضغط الدم",
     "Check in 'Vitals & Biometrics' and fill with updated measurements for 'Temperature', 'Pulse', 'R. rate', and 'Weight'": "قم بتسجيل الدخول في البيانات الحيوية والقياسات الحيوية واملأها",
+    "Consider prescription of ferrous salts": "قم باعتبار وصفة الفيلوفولك (الحديد)",
     "Contraception method prescribed": "تم البدء في تنظيم الأسرة",
     "Contraception started in this visit?": "موانع الحمل بدأت في هذه الزيارة",
+    "Copper IUD": null,
     "Cord condition": "حالة الحبل",
+    "Danger signs for mother and infant": null,
     "Date of discharge": "المستشفى وتاريخ الخروج",
     "Date of discharge from PNC": "تاريخ الخروج من الرعاية بعد الولادة",
     "Date to return to first PNC visit": "تاريخ العودة إلى زيارة PNC الأولى",
-    "Delivery ": "الولادة",
+    "Dead": null,
+    "Delivery": "الولادة",
     "Describe": "النتائج",
+    "Diabetes": null,
     "Diagnosis": "التشخيص",
+    "Emergency": null,
+    "Emergency contraceptive": null,
+    "Epilepsy": null,
+    "Episiotomy": null,
     "Evaluation of the infant 1": "تقييم الرضيع 1",
     "Evaluation of the infant 2": "تقييم الرضيع 2",
-    "Evaluation of the infant 3": "تقييم الرضيع 3",
```

Comment:
Add this back. Might need to check if it has been revised but let's add this back for now